File: QTD2.Infrastructure/Model/Task Requalification/TQReleaseByTaskAndSkillOptions.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.Task_Requalification
{
    public class TQReleaseByTaskAndSkillOptions
    {
        public int TaskId { get; set; }
        public int EnablingObjectiveId { get; set; }

        public int[] EmpIds { get; set; }

        public ICollection<EvaluatorOptions> EvaluatorOptions { get; set; } = new List<EvaluatorOptions>();

        public int? EvalMethodId { get; set; }

        public DateTime DueDate { get; set; }

        public DateTime ReleaseDate { get; set; }

        public bool OneReq { get; set; }

        public int MultiReq { get; set; }

        public bool OrderMatters { get; set; }

        public bool ShowSuggestions { get; set; }

        public bool ShowQuestions { get; set; }
    }
}

File: QTD.SPA/src/app/_DtoModels/TaskQualification/TQReleaseByTaskAndSkillOptions.ts
import { EvaluatorOption } from "./EvaluatorOptions";

export class TQReleaseByTaskAndSkillOptions{
  taskId:any;
  empIds:any[] = [];
  enablingObjectiveId:any;
  evaluatorOptions:EvaluatorOption[] = [];
  evalMethodId = null;
  dueDate!:Date;
  releaseDate!:Date;
  oneReq!:boolean;
  multiReq!:number;
  orderMatters!:boolean;
  showSuggestions!:boolean;
  showQuestions!:boolean;
}
